projects/igniteui-angular/src/lib/icon/icon.service.ts

```diff
@@ -180,7 +177,7 @@ export class IgxIconService {
      */
     public getSvgIcon(name: string, family: string = '') {
         const familyClassName = this.familyClassName(family);
-        return this._cachedSvgIcons.get(familyClassName)?.get(name);
+        return this._platformUtil.isBrowser && this._cachedSvgIcons.get(familyClassName)?.get(name);
```

Comment:
Remove this check. In platform server the method would just return `undefined`, so nothing will throw. If in the future we implement server-handling for this, then this method will return the icon properly.